### PR TITLE
Create or update baselines to match 2.0

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

**Comment by Eilon** [comment]
Why was a baseline removed?

**Comment by Tratcher** [comment]
It wasn't removed, but it was trimmed of types from the Internal namespace. Those weren't supposed to be listed anyways right?

**Comment by Eilon** [comment]
@Tratcher oh ok removing the .Internal is certainly fine. But yeah I wouldn't have thought they'd be in there to begin with. @javiercn any idea on that?

**Comment by javiercn** [comment]
We need to keep .Internal for patch releases. 2.0.0 to 2.0.1 we should not do any public API breaking change even in the .Internal namespace. For that, we need to capture .Internal in the baselines. The tool has a mode to exclude things on the .Internal namespace that can be use for 2.0->2.1 type of changes.

Does that make sense?

**Comment by Eilon** [comment]
@javiercn ah ok should we use that switch here then? Or, @ryanbrandenburg , is it being used already?

**Comment by ryanbrandenburg** [comment]
I did not set any switches when doing this, I just did the "normal" generation. @javiercn what is the switch we need to turn on for this 2.0->2.1 move?

**Comment by javiercn** [comment]
@ryanbrandenburg The normal generation should be fine. It's when checking that we should not pass --exclude-public-internal

**Comment by ryanbrandenburg** [comment]
Best guess is @BrennanConroy is the one to review this. Assign whoever is a better choice if it's not you.

**Comment by ryanbrandenburg** [comment]
The failure in appveyor is an unrelated unit test.

File: src/Microsoft.AspNetCore.Server.IntegrationTesting/baseline.netcore.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "AssemblyIdentity": "Microsoft.AspNetCore.Server.IntegrationTesting, Version=0.4.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
+  "AssemblyIdentity": "Microsoft.AspNetCore.Server.IntegrationTesting, Version=0.4.2.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Server.IntegrationTesting/baseline.netcore.json]: Should this be excluded as a prototype?

**Pull request merged**